src/catalog/catalog_entry/table_catalog_entry.cpp

```diff
@@ -115,6 +115,21 @@ unique_ptr<CatalogEntry> TableCatalogEntry::AlterEntry(ClientContext &context, A
 		return make_unique<TableCatalogEntry>(catalog, schema, (BoundCreateTableInfo *)bound_create_info.get(),
 		                                      storage);
 	}
+	case AlterTableType::RENAME_TABLE: {
+        auto rename_info = (RenameTableInfo *)table_info;
+        auto create_info = make_unique<CreateTableInfo>(schema->name, rename_info->new_table_name);
```

Comment:
This looks like duplicate code from TableCatalogEntry::Copy below. Perhaps that function could be used here instead?

Reply:
TableCatalogEntry::Copy can be used here. I will change that.